ISSUE_TEMPLATE.md

```diff
@@ -1,14 +1,14 @@
 ***PCSX2 version:***
 ``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
 
-- **Note**: It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
-
-- **Note2**: The new backwards compatibility (psx) mode is in active development. Issue reports are too early at this point.
+- **Notes**:
+  - It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
+  - The new PSX backward compatibility mode is still in its nascent state. Issue reports are too early at this point.
```

Comment:
In my opinion, ``"Nascent"`` just seems like a perfect word to be used over here. Also the intention is to prevent users from reporting PSX mode bug reports which is made clear by the statement following it  (``Issue reports are too early at this point``).